connectors/services/__init__.py

```diff
@@ -3,7 +3,10 @@
 # or more contributor license agreements. Licensed under the Elastic License 2.0;
 # you may not use this file except in compliance with the Elastic License 2.0.
 #
+# fmt: off
+from connectors.services.access_control_sync_job_execution import AccessControlSyncJobExecutionService  # NOQA
```

Comment:
Why NOQA?

Reply:
`# NOQA` is to skip line check from [ruff](https://pypi.org/project/ruff/).

They are present in the original code. If I don't include them for the two new lines, they will be removed after running `make autoformat`

`# fmt: off/on` is to skip line check from [black](https://pypi.org/project/black/).